releasenotes-builder/src/test/java/com/github/checkstyle/templates/TemplateProcessorTest.java

```diff
@@ -306,5 +307,10 @@ private void createAllCommits() {
             + "LongAndWeExpectItToGetWrappedDueToBeingTooLng", "Author 11");
         addCommit("Issue #9: Mock issue title 9 escape @ and @@@@@", "Author 12");
         addCommit("Issue #10: Mock issue title 10 escape < > & <&<>&<<", "Author 13");
+        addCommit("Issue 11: Mock issue title 11 missing #", "Author 14");
+        addCommit("Pull 12: Mock pull title 12 missing #", "Author 15");
+        addCommit("miscasmdifs324y23123sda!@.asd12", "Author 16");
+        addCommit("Issue #asd: asd", "Author 17");
+        addCommit("Pull #13: Mock pull title 13", "Author 18");
```

Comment:
Please add a test case that shows the `^` is needed for the regular expression you changed.

Reply:
Done.
`.Issue #14: Mock issue title 14`